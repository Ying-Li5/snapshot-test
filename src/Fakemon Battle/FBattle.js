import React from "react";
import Card from "react-bootstrap/Card"
import FPic from './FPic.jpg'

function FBattleCard() {
    return (
        <div className="card">
          <Card style={{ width: "18rem"}}>
          <Card.Img variant="top" src={FPic} />
          
          <Card.Body>
              <Card.Title>Fakemon Battle</Card.Title>
              <Card.Text>
              Fakèmon Battle was created for my milestone projects for people to see my skills 
              and knowledge built during Part 1 of my Bootcamp studies, it was inspired from 
              playing Pokèmon while growing up
              </Card.Text>
          </Card.Body>
          </Card>
        </div>
    );
  }
  
  export default FBattleCard