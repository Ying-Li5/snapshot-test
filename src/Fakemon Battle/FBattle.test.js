import FBattle from './FBattle'
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const tree = renderer.create(<FBattle />).toJSON()
  expect(tree).toMatchSnapshot()
})