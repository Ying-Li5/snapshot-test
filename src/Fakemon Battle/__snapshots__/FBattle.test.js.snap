// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="FPic.jpg"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Fakemon Battle
      </div>
      <p
        className="card-text"
      >
        Fakèmon Battle was created for my milestone projects for people to see my skills and knowledge built during Part 1 of my Bootcamp studies, it was inspired from playing Pokèmon while growing up
      </p>
    </div>
  </div>
</div>
`;
